src/mrpro/data/traj_calculators/KTrajectoryRadial2D.py

```diff
@@ -46,3 +46,32 @@ def __call__(self, kheader: KHeader) -> KTrajectory:
         kz = torch.zeros(1, 1, 1, 1)
 
         return KTrajectory(kz, ky, kx)
+
+    def create_manually(self, n_spokes: int = 240, n_k0: int = 256, initial_angle: float = 0) -> KTrajectory:
```

Comment:
I think I would prefer it if we would give krad:torch.Tensor and kang_index:torch.Tensor as input to this function. Then we could replace the code in `__call__` with this function.

Reply:
or say krad: torch.Tensor | int and if it is int then you calculate it assuming symmetric sampling and something similar for kang_index